auth/handlers/handle_authorized-clients.go

```diff
@@ -69,19 +70,39 @@ func (h *AuthHandlers) clientsProc(req *request.AuthReq) error {
 	return nil
 }
 
-func (h *AuthHandlers) getAuthorizedClients(req *request.AuthReq) (ss authClients, err error) {
+func (h *AuthHandlers) unAuthorizeClientsByUser(req *request.AuthReq) error {
+	userID, err := strconv.ParseUint(req.Request.PostFormValue("userID"), 10, 64)
+	if userID == 0 {
+		return err
+	}
+
+	ss, err := h.getAuthorizedClients(req.Context(), userID)
+	if err != nil {
+		return err
+	}
+
+	for _, s := range ss {
+		clientID := s.ID
+		if err = h.TokenService.DeleteByID(req.Context(), clientID); err != nil {
```

Comment:
how about if we just use DeleteByUserID() instead of loading+looping+deleting-each-one?